sources/packages/backend/apps/db-migrations/src/sql/DisbursementSchedules/Add-tuition-remittance-effective-amount-col.sql

```diff
@@ -0,0 +1,7 @@
+-- Add tuition_remittance_effective_amount to disbursement_schedules table.
+ALTER TABLE
+    sims.disbursement_schedules
+ADD
+    COLUMN IF NOT EXISTS tuition_remittance_effective_amount INT NOT NULL DEFAULT 0;
```

Comment:
I would say that the column should be null until some calculation happens, which can be never if the disbursement never happens.

Reply:
May be not a part of the PR or may be. The effective_amount and the awards(which are money values) are NUMERIC(8 , 2) but the tuition_remittance and tuition_remittance_effective_value are INT. Don't we eventually have one data type for the the amounts? @ann-aot @andrewsignori-aot @guru-aot @andrepestana-aot @sh16011993 

![image](https://user-images.githubusercontent.com/54600590/228660744-7d722871-3472-4e1e-b3bb-33e9f6d8ca1d.png)